app/components/public/stream/side-panel.hbs

```diff
@@ -1,10 +1,12 @@
-<button {{did-insert this.setup}} class="ui icon button" style="position: absolute; top: 0; margin: 10px" {{action (mut this.shown) true}}>
-  <i class="arrow right icon"></i>
+<button {{did-insert this.setup}} class="ui icon small button toggle-btn" style="position: absolute; top: 0; margin-top: 10px; left: -5px" {{action (mut this.shown) true}}>
```

Comment:
move styling to toggle-btn class

Reply:
toggle btn has it's own defination of styling moving inline stlying to toggle btn class will break the styling for the right side button of side panel.

Reply:
have you tried it? try once, if it didn't work create new class move style there.